src/nspawn/nspawn-seccomp.h

```diff
@@ -21,4 +21,23 @@
 
 #include <sys/types.h>
 
-int setup_seccomp(uint64_t cap_list_retain);
+#ifdef HAVE_SECCOMP
+#include "seccomp-util.h"
+
+typedef struct SyscallFilterSetCap {
+        SyscallFilterSet filter;
+        int capability;
+} SyscallFilterSetCap;
+#endif
```

Comment:
Well, I thought that seccomp-util.h can be included only when having seccomp headers, but that's not true, so I removed it. Thanks for pointing out.